docker_test.go

```diff
@@ -1024,7 +1024,7 @@ func Test_BuildContainerFromDockerfile(t *testing.T) {
 	})
 
 	t.Log("pinging redis")
-	pong, err := client.Ping().Result()
+	pong, err := client.Ping(ctx).Result()
```

Comment:
Mmm it's weird the the docs are updated since https://github.com/testcontainers/testcontainers-go/commit/25fc53e2585109a09188824c41b7c90b09716c93

Docs/Web: https://golang.testcontainers.org/examples/redis/